augur/tasks/git/dependency_tasks/dependency_util/dependency_calculator.py

```diff
@@ -20,6 +23,9 @@ def get_dependency_analysis_module_tuples():
 	yield java_deps, 'java'
 	yield cpp_deps, 'C++'
 	yield c_deps, 'C'
+    yield go_deps, 'go'
```

Comment:
It looks like there is an indentation error for these three lines that need to be corrected.